pkg/tapo/client.go

```diff
@@ -291,7 +291,7 @@ func dial(req *http.Request, brand, username, password string) (net.Conn, *http.
 	if err != nil {
 		return nil, nil, err
 	}
-	_ = res.Body.Close() // ignore response body
+	_, _ = io.Copy(io.Discard, res.Body) // ignore response body
```

Comment:
This is great, but you still need to close the body
```suggestion
	_, _ = io.Copy(io.Discard, res.Body) // discard leftovers
	_ = res.Body.Close() // ignore response body
```

Reply:
Thanks for the clarification and suggestion.
Still works after the change, so added it to this PR.